packages/evm/contracts/adapters/Electron/lib/Ownable.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
same here...you can import `Ownable` from oz.

Reply:
It's done